### ISSUE TITLE
"Title", "Genre" and "Rating" fields are required but screen reader is not announcing as required: A11y_ASP.NET_Movies_Create Movies_ScreenReader

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/aspnetcore#56284]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #BM_ASP.NET_Web_Jun2024; #.NETCore; #WCAG1.3.1; #DesktopWeb; #Win11; #FTP; #Narrator; #NVDA; #Element:EditFields; #ChromiumEdge;

Environment Details:
App name: ASP.NET
Window Version: Win 11
OS Build: 22621.3085
Screen readers: Narrator, NVDA 2023.3.3

Repro Steps:
Turn on Narrator/NVDA.
Download the "BlazorWebApp" zip file and run .exe file.
Command prompt will open, TAB to "http://localhost:5000/" link and press CTRL + ENTER key.
TAB to "Movies" tab and hit ENTER key. TAB to "Create New" button and hit ENTER key.
TAB to "Title", "Genre" and "Rating" fields and observe that Narrator screen reader is not announcing these fields as required
Actual Result:
"Title", "Genre" and "Rating" fields are required but Narrator screen reader is not announcing as required. Also * is not provided for required fields.

Similar issue also observed with NVDA.

Expected Result:
"Title", "Genre" and "Rating" fields are required hence Narrator screen reader should be announcing as required. Also * should be provided for required fields.

User Impact:
Screen reader users will not be able to get information about the required fields and will miss out the information.

Attachment

**Comment by Yash14j** [comment]
GithubTags:#Rev:yaja;

**Comment by PoojaNamde** [comment]
#Regressed:09-20-24;

Verified the bug in the environment http://localhost:5000/movies/create
still able to repro it. hence reactivating it.
Please find repro attachment for reference.

[Video 9-20 at 10.48.webm](https://github.com/user-attachments/assets/6b662939-6868-42d3-8dc3-ead0c2109132)

**Comment by PoojaNamde** [comment]
Verified the bug in the environment https://localhost:7202/Movies/[Create](https://localhost:7202/Movies/Create)
still able to repro it. hence reactivating it.
Please find repro attachment for reference.


![Image](https://github.com/user-attachments/assets/79cb28fd-4718-4ed9-b58d-a937e7f2d7a1)
